### ISSUE TITLE
web3-providers-ws install issue in Windows

### ISSUE THREAD

**Issue by EthyMoney** [ethereum/web3.js#2999]
### Description 

Installing web3 module in Windows environment fails at install script for web3-providers-ws due to a remove command issue.

#### Expected behavior
del should be used when executing remove commands in Windows.

#### Actual behavior
rm is being used to remove in Windows.

#### Steps to reproduce the behavior

Install web3 npm module normally while in Windows environment.

#### Error Logs 

```npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! web3-providers-ws@1.2.0 preinstall: `rm -rf node_modules/websocket/.git`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the web3-providers-ws@1.2.0 preinstall script.

```

### Versions 
* web3.js: 1.2.0
* nodejs: 12.7.0
* npm: 6.10.2
* environment: Windows 10 pro x64 build 1903

**Comment by deamme** [comment]
Same issue here.. It should be using https://github.com/isaacs/rimraf if possible

**Comment by nivida** [comment]
duplication of https://github.com/ethereum/web3.js/issues/2971 and will be released with [1.2.1](https://github.com/ethereum/web3.js/blob/1.x/CHANGELOG.md)
